squirrel_sensing_node/tactile_calibration.ini

```diff
@@ -1,14 +1,14 @@
 #file containing voltage dividers for tactile and proximity sensors, one number per line
-#following are voltage divider for tactile sensor (calibration for TUW hand). F1-S2 F2-S1 F3-S3
-1.3
-1.9
-3.2
-2.2
-3
-2.2
-1.9
-2.45
-3.7
+#following are voltage divider for tactile sensor (KCL hand)
```

Comment:
is this right? should we not keep the  version from Liza https://github.com/squirrel-project/squirrel_driver/commit/261bb3f3869c41d3c94e283e61cc3d061f42a526

Reply:
We also test the sensors, i think it's better to keep the version from liza